docs/reference/content/guides/querying.md

```diff
@@ -0,0 +1,250 @@
+have a [text index]({{< docsref "/core/index-text/" >}}) defined first.  Using Morphia that definition would look like this:
+
+```java
+@Indexes(@Index(fields = @Field(value = "$**", type = IndexType.TEXT)))
+public static class Greeting {
+    @Id
+    private ObjectId id;
+    private String value;
+    private String language;
+    
+    ...
+}
+```
+
+The `$**` value tells MongoDB to create a text index on all the text fields in a document.  A more targeted index can be created, if 
+desired, by explicitly listing which fields to index.  Once the index is defined, we can start querying against it like this [test]({{< 
+srcref "morphia/src/test/java/org/mongodb/morphia/query/TestTextSearching.java">}}) does:
+
+```java
+morphia.map(Greeting.class);
+datastore.ensureIndexes();
+
+datastore.save(new Greeting("good morning", "english"));
+datastore.save(new Greeting("good afternoon", "english"));
+datastore.save(new Greeting("good night", "english"));
+datastore.save(new Greeting("good riddance", "english"));
+
+datastore.save(new Greeting("guten Morgen", "german"));
+datastore.save(new Greeting("guten Tag", "german"));
+datastore.save(new Greeting("guten Nacht", "german"));
+
+datastore.save(new Greeting("buenos días", "spanish"));
+datastore.save(new Greeting("buenas tardes", "spanish"));
+datastore.save(new Greeting("buenos noches", "spanish"));
+
+List<Greeting> good = datastore.createQuery(Greeting.class)
+                             .search("good")
+                             .order("_id")
+                             .asList();
+Assert.assertEquals(4, good.size());
+Assert.assertEquals("good morning", good.get(0).value);
+Assert.assertEquals("good afternoon", good.get(1).value);
+Assert.assertEquals("good night", good.get(2).value);
+Assert.assertEquals("good riddance", good.get(3).value);
+```
+
+As you can see here, we create `Greeting` objects for multiple languages.  In our test query, we're looking for occurrences of the work 
+"good" in any document.  We created four such documents and our query returns exactly those four.
+
+## Other Query Options
+
+There is more to querying than simply filtering against different document values.  Listed below are some of the options for modifying 
+the query results in different ways.
+
+### Projections
+
+[Projections]({{< docsref "reference/method/db.collection.find/#projections" >}}) allow you to return only a subset of the fields in a 
+document.  This is useful when you need to only return a smaller view of a larger object.  Borrowing from the [unit tests]({{< srcref 
+"morphia/src/test/java/org/mongodb/morphia/TestQuery.java" >}}), this is an example of this feature in action:
+
+```java
+ContainsRenamedFields user = new ContainsRenamedFields("Frank", "Zappa");
+getDs().save(user);
+
+ContainsRenamedFields found = getDs().find(ContainsRenamedFields.class).retrievedFields(true, "first_name").get();
```

Comment:
The retrievedFields method is the key thing here, but it's buried amidst the rest of the code.  It's also not clear how to retrieve more than one field.  Does it take varargs?  The first argument also deserves an explanation.